sechub-sereco/src/test/java/com/daimler/sechub/sereco/importer/SarifV1JSONImporterTest.java

```diff
@@ -0,0 +1,170 @@
+package com.daimler.sechub.sereco.importer;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.daimler.sechub.sereco.ImportParameter;
+import com.daimler.sechub.sereco.metadata.SerecoCodeCallStackElement;
+import com.daimler.sechub.sereco.metadata.SerecoMetaData;
+import com.daimler.sechub.sereco.metadata.SerecoSeverity;
+import com.daimler.sechub.sereco.metadata.SerecoVulnerability;
+import com.daimler.sechub.sereco.test.SerecoTestFileSupport;
+
+public class SarifV1JSONImporterTest {
+
+    private static SarifV1JSONImporter importerToTest;
+    private static String sarifBrakeman;
+    private static String sarifEmptyResult;
+    private static String sarifThreadflowsExample;
+    
+    @BeforeClass
+    public static void before() {
+        importerToTest = new SarifV1JSONImporter();
```

Comment:
Please have a look into `ImporterRegistry.java`. All importers are created and stored in a `importers` list. Therefore they are reused during runtime and this would match the behavior in this test in some way.

I believe side effects are no problem, or even would be of interest for us if they occur early within our UTs because this would hint to bigger problems with the actual implementation.